test/netlib/http/test_cookies.py

```diff
@@ -260,9 +280,29 @@ def test_is_expired():
     # or both
     assert cookies.is_expired(CA([("Expires", "Thu, 01-Jan-1970 00:00:00 GMT"), ("Max-Age", "0")]))
 
-    assert not cookies.is_expired(CA([("Expires", "Thu, 24-Aug-2063 00:00:00 GMT")]))
+    assert not cookies.is_expired(CA([("Expires", "Fri, 24-Aug-2063 00:00:00 GMT")]))
     assert not cookies.is_expired(CA([("Max-Age", "1")]))
-    assert not cookies.is_expired(CA([("Expires", "Thu, 15-Jul-2068 00:00:00 GMT"), ("Max-Age", "1")]))
+    assert not cookies.is_expired(CA([("Expires", "Sun, 15-Jul-2068 00:00:00 GMT"), ("Max-Age", "1")]))
 
     assert not cookies.is_expired(CA([("Max-Age", "nan")]))
     assert not cookies.is_expired(CA([("Expires", "false")]))
+
+
+def test_group_cookies():
+    def group(cookie):
+        return cookies.group_cookies(cookies.parse_cookie_header(c))
+
+    c = "one=uno; foo=bar; foo=baz"
+    assert len(group(c)) == 3
+
+    c = "one=uno; Path=/; foo=bar; Max-Age=0; foo=baz; expires=24-08-1993"
+    assert len(group(c)) == 3
+
+    c = "one=uno;"
+    assert len(group(c)) == 1
+
+    c = "one=uno; Path=/; Max-Age=0; Expires=24-08-1993"
+    assert len(group(c)) == 1
```

Comment:
I think we should assert the groups explicitly - not only the length of the result.